Skin Object scaffording not working

## Describe the bug
The Upendo Ventures DNN project generator for creating Skin Objects is throwing a javascript error.

events.js:167
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module 'uuid'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:594:15)
    at Function.Module._load (internal/modules/cjs/loader.js:520:25)
    at Module.require (internal/modules/cjs/loader.js:650:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (C:\Users\****\AppData\Roaming\npm\node_modules\generator-upendodnn\generators\skinobject\index.js:4:14)
    at Module._compile (internal/modules/cjs/loader.js:702:30)    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
Emitted 'error' event at:
    at Immediate.setImmediate (C:\Users\***\AppData\Roaming\npm\node_modules\generator-upendodnn\node_modules\yeoman-generator\lib\index.js:433:22)
    at runCallback (timers.js:696:18)
    at tryOnImmediate (timers.js:667:5)    at processImmediate (timers.js:649:5)

## To Reproduce
Steps to reproduce the behavior:
1. Run: yo upendodnn
2. Select the Skin Object project
3. Add local URL and path
4. See error

## Expected behavior
Project is created just like the other projects which are working.
## Screenshots


## Errors

events.js:167
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module 'uuid'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:594:15)
    at Function.Module._load (internal/modules/cjs/loader.js:520:25)
    at Module.require (internal/modules/cjs/loader.js:650:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (C:\Users\****\AppData\Roaming\npm\node_modules\generator-upendodnn\generators\skinobject\index.js:4:14)
    at Module._compile (internal/modules/cjs/loader.js:702:30)    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
Emitted 'error' event at:
    at Immediate.setImmediate (C:\Users\***\AppData\Roaming\npm\node_modules\generator-upendodnn\node_modules\yeoman-generator\lib\index.js:433:22)
    at runCallback (timers.js:696:18)
    at tryOnImmediate (timers.js:667:5)    at processImmediate (timers.js:649:5)

## Additional context
Add any other context about the problem here.

It looks like this may be the culprit ("Error: Cannot find module 'uuid'):
https://github.com/UpendoVentures/generator-upendodnn/blob/eb576e5d4939d06ad872a20531805caa20bbc0ea/generators/skinobject/index.js#L4

--- a/generators/skinobject/index.js
+++ b/generators/skinobject/index.js
@@ -1,7 +1,7 @@
 'use strict';
 const DnnGeneratorBase = require('../lib/DnnGeneratorBase');
 const chalk = require('chalk');
-const uuid = require('uuid');
+const uuid = require('uuid-v4');
 
 module.exports = class extends DnnGeneratorBase {
   constructor(args, opts) {
@@ -66,7 +66,7 @@ module.exports = class extends DnnGeneratorBase {
 
     var msBuildVersion = this._getMsBuildVersion();
 
-    if (msBuildVersion == ""){
+    if (msBuildVersion == "") {
       this.log(chalk.red("YIKES! A valid version of MSBuild was not found! This is a critical error... :("));
     }
 
@@ -88,9 +88,9 @@ module.exports = class extends DnnGeneratorBase {
 
   writing() {
     this.log(chalk.white('Creating MVC Module.'));
-	
-	// mod: this follows the Upendo development/solution pattern
-	this.destinationRoot("SkinObjects/");
+
+    // mod: this follows the Upendo development/solution pattern
+    this.destinationRoot("SkinObjects/");
 
     let namespace = this.props.namespace;
     let extensionName = this.props.extensionName;
@@ -209,7 +209,7 @@ module.exports = class extends DnnGeneratorBase {
     this.fs.extendJSON(this.destinationPath(extensionName + '/package.json'), pkgJson);
   }
 
-  install() {}
+  install() { }
 
   end() {
     this.log(chalk.white('Installed Skin Object npm Dependencies.'));

